Sources/SwiftDocC/SwiftDocC.docc/Resources/IndexingRecords.spec.json

```diff
@@ -53,6 +53,12 @@
                     },
                     "rawIndexableTextContent": {
                         "type": "string"
+                    },
```

Comment:
Should we be bumping the minor version of the spec as part of this?

Reply:
Either the minor or the patch component. I can never remember what's expected for semantic versions before the 1.0 release. https://semver.org only says that anything can change before the 1.0 release but doesn't offer any 0.minor.patch specific guidance about which version component to update.
 
This is a new feature to the spec but it's also backwards compatible.

Reply:
We seem to have only ever bumped the minor version elsewhere so I bumped the minor version here as well.